plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/session/SessionState.kt

```diff
@@ -3,10 +3,13 @@
 
 package software.aws.toolkits.jetbrains.services.amazonqFeatureDev.session
 
+import software.aws.toolkits.jetbrains.services.amazonqFeatureDev.util.FeatureDevClientUtil
+
 interface SessionState {
     val tabID: String
     val phase: SessionStatePhase?
     var approach: String
+    val featureDevClientUtil: FeatureDevClientUtil
```

Comment:
Should we define this `featureDevClientUtil` in SessionStateConfig instead; attaching this with `SessionState` here makes it a bit misleading.

Reply:
Yeah, as discussed offline, renamed it to be FeatureDevService and moved the parameter to the config.